### PR TITLE
Resolve (some) browser warnings

### PR DESCRIPTION
**Pull request by kayserifserif**

### CODE REVIEW

**Review by mark-fathom — CHANGES_REQUESTED** [review]
Definitely want to keep the MD fonts in there. Although we can't make them available to the general public, they are the primary fonts we use in the version of the app that we publish.

**Comment by kayserifserif** [comment]
Ahhh. Thanks for the check! Reverted that commit.

File: src/ts/ui/mutations/mutationprevalencecanvas.ts [hunk 1]
```diff
@@ -159,7 +159,7 @@ export class MutationPrevalenceCanvas {
       } else {
         console.debug("Could not load MD fonts, using Roboto fallbacks");
         const backupPromises = Promise.all([
-          (new FontFace("Roboto", "url('/assets/fonts/roboto/roboto-medium.woff2')", {weight: '500', style: 'normal'})).load(),
+          (new FontFace("Roboto", "url('/assets/fonts/roboto/roboto-medium.ttf')", {weight: '500', style: 'normal'})).load(),
```
[the remark this answers is not in the document]
Comment (kayserifserif) [reply src/ts/ui/mutations/mutationprevalencecanvas.ts:162]: For some reason, we only have the ttf for medium and bold weights. Other weights have the woff2. Was there any reason for that?

[the remark this answers is not in the document]
Comment (mark-fathom) [reply src/ts/ui/mutations/mutationprevalencecanvas.ts:162]: If there was, I doubt it was a good one.

[the remark this answers is not in the document]
Comment (mark-fathom) [reply src/ts/ui/mutations/mutationprevalencecanvas.ts:162]: I stand corrected: it looks like jspdf only works well with ttf fonts. In fact, we had to make a custom conversion of .otf to .ttf fonts to get it to work.

**Pull request merged**